deeplearning4j-scaleout/deeplearning4j-aws/pom.xml

```diff
@@ -46,6 +46,11 @@
                         <artifactId>jsch</artifactId>
                         <version>0.1.51</version>
                 </dependency>
+            <dependency>
+                <groupId>org.apache.spark</groupId>
+                <artifactId>spark-core_2.11</artifactId>
+                <version>2.1.0</version>
```

Comment:
Don't we want a ${spark.version} here? That or at least use an inherited version somewhere? Would rather not have multiple spark version declarations.